client-app/tsconfig.json

```diff
@@ -1,20 +1,37 @@
 {
     "compilerOptions": {
+        "jsx": "react",
         /* Visit https://aka.ms/tsconfig to read more about this file */
 
         /* Language and Environment */
-        "target": "es2016" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
+        // "target": "es2016" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
 
-        /* Modules */
-        "module": "commonjs" /* Specify what module code is generated. */,
-        "esModuleInterop": true /* Emit additional JavaScript to ease support for importing CommonJS modules. This enables 'allowSyntheticDefaultImports' for type compatibility. */,
-        // "preserveSymlinks": true,                         /* Disable resolving symlinks to their realpath. This correlates to the same flag in node. */
-        "forceConsistentCasingInFileNames": true /* Ensure that casing is correct in imports. */,
+        // /* Modules */
+        // "module": "commonjs" /* Specify what module code is generated. */,
+        // "esModuleInterop": true /* Emit additional JavaScript to ease support for importing CommonJS modules. This enables 'allowSyntheticDefaultImports' for type compatibility. */,
+        // // "preserveSymlinks": true,                         /* Disable resolving symlinks to their realpath. This correlates to the same flag in node. */
+        // "forceConsistentCasingInFileNames": true /* Ensure that casing is correct in imports. */,
 
-        /* Type Checking */
-        "strict": true /* Enable all strict type-checking options. */,
-        "skipLibCheck": true /* Skip type checking all .d.ts files. */,
-        "jsx": "react-jsx"
+        // /* Type Checking */
+        // "strict": true /* Enable all strict type-checking options. */,
+        // "skipLibCheck": true /* Skip type checking all .d.ts files. */,
+        // "jsx": "react-jsx",
+
+        "target": "es5",
```

Comment:
Did you have any plan to change target from es2016 to es5? Is this all new configs necessary?

Reply:
I found this advantages: Strict mode, which allows developers to opt into a restricted variant of JavaScript 
JSON support, which streamlines data interchange in web applications 
New methods for arrays and strings, including forEach, map, filter, and reduce

Reply:
@miabu-pashh strict mode was already true. I would suggest to revert this config file entirely. Please create a new PR for such updates in future